### ISSUE TITLE
Pod Controller not accounting for :ejectPod()

### ISSUE THREAD

**Issue by deltamolfar** [wiremod/wire#3017]
**Describe the bug**
Controler Pod -> CP
When ejecting a player from CP-linked vehicle using E2 :ejectPod() method, the "Active", "Driver" and other outputs of CP isn't being refreshed to represent the absence of a driver.

**To Reproduce**
1. Spawn a vehicle, CP, and link them
2. Place E2 with this code:
```lua
event playerEnteredVehicle(Player:entity, Vehicle:entity) {
    Vehicle:ejectPod()
}
```
3. Press debug on Pod Controller, or get it outputs in any other way
4. Sit in the vehicle, get ejected, look at outputs

**Expected behavior**
The outputs should be reset in the same way as when you ejecting a driver using CP's eject input/when driver exits a vehicle himself

**Screenshots**
![image](https://github.com/wiremod/wire/assets/72973198/2f9a09e4-a098-4ec7-8a13-522c82c2c0a5)

**Comment by deltamolfar** [comment]
Checked the code for quick fix, never found it.

ENT:PlayerExited() is executing fine in terms the hook, and all if statements are as expected.
The problematic part as far as I see is that WireLib.TriggerOutput have no effect. But as I'm unfamiliar with wiremo code base, it's hard for me to go further

**Comment by LeoLongtail** [comment]
I don't think this bug happens anymore. The outputs show no driver and active=0
